core/src/main/java/com/squarespace/template/plugins/platform/CommerceFormatters.java

```diff
@@ -493,14 +493,14 @@ public void apply(Context ctx, Arguments args, Variables variables) throws CodeE
       JsonNode pricingOptions = CommerceUtils.getPricingOptionsAmongLowestVariant(node);
 
       if (pricingOptions != null && pricingOptions.size() > 0) {
-        if (CommerceUtils.hasVariants(node)) {
+        if (CommerceUtils.hasVariedPrices(node)) {
           // This will return either salePriceMoney or priceMoney depending on whether the onSale is true or false.
           // That's because this block here is the from {price} so the from price needs to be the lowest possible price
           // taking into if a variant is onSale.
           JsonNode subscriptionFromPricingNode = CommerceUtils.getSubscriptionMoneyFromFirstPricingOptions(pricingOptions);
 
           subscriptionResults.put("fromText", StringUtils.defaultIfEmpty(
-                  ctx.resolve(Constants.PRODUCT_PRICE_FROM_TEXT_KEY).asText(), "from {price}"));
+                  ctx.resolve(Constants.PRODUCT_PRICE_FROM_TEXT_KEY).asText(), "from {fromPrice}"));
```

Comment:
because `Constants.PRODUCT_PRICE_FROM_TEXT_KEY` is pulled from `localizedStrings` from NBF, NBF expects the variable `fromPrice` so we have to keep it consistent or else the price won't show in NBF.